src/MoBi.Presentation/DTO/BreadCrumbsDTO.cs

```diff
@@ -53,5 +56,36 @@ public string PathElementByIndex(int index)
       {
          return _elementList.Count > index ? _elementList[index] : string.Empty;
       }
+
+      private static class AllRules
+      {
+         private static bool elementDoesNotContainWildcardCharacters(string element)
+         {
+            if (string.IsNullOrEmpty(element))
+               return true;
+
+            return !element.Contains(Constants.WILD_CARD);
+         }
+
+         private static IBusinessRule elementCannotContainIllegalCharacters(Expression<Func<BreadCrumbsDTO<T>, string>> expression) => CreateRule.For<BreadCrumbsDTO<T>>()
+            .Property(expression)
+            .WithRule((dto, element) => elementDoesNotContainWildcardCharacters(element))
+            .WithError(AppConstants.PathCannotContainWildcardCharacters(new[] { Constants.WILD_CARD }));
+
+         public static IReadOnlyList<IBusinessRule> All { get; } = new[]
+         {
+            elementCannotContainIllegalCharacters(x => x.PathElement0),
+            elementCannotContainIllegalCharacters(x => x.PathElement1),
+            elementCannotContainIllegalCharacters(x => x.PathElement2),
+            elementCannotContainIllegalCharacters(x => x.PathElement3),
+            elementCannotContainIllegalCharacters(x => x.PathElement4),
+            elementCannotContainIllegalCharacters(x => x.PathElement5),
+            elementCannotContainIllegalCharacters(x => x.PathElement6),
+            elementCannotContainIllegalCharacters(x => x.PathElement7),
+            elementCannotContainIllegalCharacters(x => x.PathElement8),
+            elementCannotContainIllegalCharacters(x => x.PathElement9)
+         };
+
+      }
```

Comment:
we have another illegal characters (like the path separator `|` ). 
I wonder why the wildcard-check is not combined with those?